### DISCUSSION TITLE
【温馨提示】各位旅行者，开拓者，绳匠们 该睡觉了

### DISCUSSION THREAD

**Discussion by opqphp** [share121/inter-knot#3324] [其他]
胡桃厨提醒您  早睡身体好 晚睡人会飘
![20240625_034059](https://github.com/user-attachments/assets/2b4a3869-01d5-4144-85ed-0888839154a3)

![illust_111446009_20240819_004149](https://github.com/user-attachments/assets/5479d2d1-21c1-404c-a388-1c4127226dde)
<img src="https://img-static.mihoyo.com/communityweb/upload/1911ab16b4af46252dbd90fc539d4fc5.png" alt="刻晴-晚安" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/a7b34f7d13a20b8c50015367b3d56fa9.png" alt="甘雨-打瞌睡" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/3b8cf75f15864abfc9eeb1b9316d9224.png" alt="胡桃-卖萌" width="75" >

**Comment by 21Seven** [comment]
才几点啊！就睡了

**Comment by Fireworks-HH** [comment]
太欢愉看，不睡

**Comment by clcl-maker** [comment]
谢谢你，我快飘了，熬夜熬的头晕

**Comment by zzztjs** [comment]
感谢提醒，好人一生平胸<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by aynami231** [comment]
😀
